bin/rspecq

```diff
@@ -65,6 +66,11 @@ OptionParser.new do |o|
     opts[:report_timeout] = v
   end
 
+  o.on("--max-requeues N", Integer, "Sets the number of max requeues to N " \
+       "times (default: #{DEFAULT_MAX_REQUEUES}).") do |v|
+    opts[:max_requeues] = v
```

Comment:
you have a bad indentation over there

Reply:
@roseliux I don't think I see it, it seems in line with the indentation of the other options above it

Reply:
ohhh sorry my bad, you are right.